docs/contributing/documentation.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to the documentation
+
+We welcome contributions to the Tracee documentation.
+
+Our documentation aims to follow the [Diátaxis documentation framework](https://diataxis.fr/) which includes:
+
+1. A getting started section -- Installation Guidelines and simple scenarios
+2. Tutorials -- End-to-end tutorials that have real-world use cases
+3. Reference Material -- How to use Tracee
+4. Contribution Guidelines
+
+To contribute to the documentation please
+
+1. Clone the Tracee GitHub repository
+2. Make changes in the `./docs` directory in the root folder
+3. Remember to edit the index in `mkdocs.yml` in the root folder if you add or move existing files or directories
+4. Test your changes to the documentation
+5. Submit a PR
+
+## Test Documentation Changes
+
+You can test your changes to the documentation by building and running a docker container.
+
+Prerequisites: Docker installed and running locally.
+
+Build the container image:
```

Comment:
The section below should be about the makefile.

```
        @echo ""
        @echo "To build the mkdocs docker container:"
        @echo ""
        @echo "    $$ make -f builder/Makefile.mkdocs mkdocs-build"
        @echo ""
        @echo "To serve mkdocs through mkdocks container:"
        @echo ""
        @echo "    $$ make -f builder/Makefile.mkdocs mkdocs-serve"
        @echo ""
        @echo "Or simply:"
        @echo ""
        @echo "    $$ make -f builder/Makefile.mkdocs"
        @echo ""
```

Reply:
Those commands don't work for me:

```bash
❯ make -f builder/Makefile.mkdocs mkdocs-build
if [ $? -ne 0 ]; then
/bin/sh: -c: line 1: syntax error: unexpected end of file
make: *** [.check_docker] Error 2
```

Reply:
Odd...

```
❯ make -f builder/Makefile.mkdocs mkdocs-build 
docker build \
	-f builder/Dockerfile.mkdocs \
	-t tracee-mkdocs:latest \
	.
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            Install the buildx component to build images with BuildKit:
            https://docs.docker.com/go/buildx/

Sending build context to Docker daemon  157.3MB
Step 1/2 : FROM squidfunk/mkdocs-material:8.3.0
 ---> 585d64194fff
Step 2/2 : RUN pip install --upgrade pip &&     pip install mike &&     pip install mkdocs-macros-plugin
 ---> Using cache
 ---> 228b9dad3ceb
Successfully built 228b9dad3ceb
Successfully tagged tracee-mkdocs:latest
```

Could be your `sh` or `make` version?

```
❯ make --version
GNU Make 4.4.1
Built for x86_64-pc-linux-gnu
...
```

```
❯ sh --version
GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)
...
```

Reply:
aha but the similar commands for Trivy used to work before we switched there to mage

> ❯ make --version
GNU Make 3.81

> ❯ sh --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin22)

Reply:
@AnaisUrlichs are the commands you provided below work?
If so, can we merge this PR?

Reply:
the docker commands work for me, the make commands do not